commons/ava_testsuite/byzantine_gecko_network_tests.go

```diff
@@ -0,0 +1,115 @@
+package ava_testsuite
+
```

Comment:
Rather than calling this file `byzantine_gecko_network_tests` (which runs the risk of becoming a mega-file like we had for the other file), let's make this one file per test = `staking_network_unrequested_chit_spammer_test.go` or something